lib/client.js

```diff
@@ -247,7 +250,7 @@ var Client = function(options) {
     return this; // shortcut handshake
   }
 
-  Tracking.setup(this);
+  tracker.setup();
```

Comment:
nit: I wonder if it's better to bring the initialization on line 225 down here, so that they're in the same spot